### PR TITLE
Map InGameHud and its inner class

### PR DESCRIPTION
**Pull request by enbrain**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 1]
```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
+	COMMENT Responsible for rendering the HUD elements while the player is in game.
+	COMMENT
+	COMMENT <p>The current instance used by the client can be obtained by {@code
+	COMMENT MinecraftClient.getInstance().inGameHud}.
```
Comment (liach) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping]:
```suggestion
	COMMENT <p>The current instance used by the client can be obtained by {@link
	COMMENT MinecraftClient#inGameHud MinecraftClient.getInstance().inGameHud}.
```
Makes this a link that goes to the actual field.

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 2]
```diff
@@ -93,13 +99,15 @@ CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
 		ARG 1 tickDelta
 		ARG 2 matrices
 	METHOD method_1760 renderStatusBars (Lnet/minecraft/class_4587;)V
+		COMMENT Renders the armor, health, hunger and oxygen bars.
```
Comment (haykam821) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping]:
```suggestion
		COMMENT Renders the armor, health, air, and hunger bars.
```

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 3]
```diff
@@ -126,3 +134,35 @@ CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
...
+		ARG 2 player
+		ARG 3 x
+		ARG 4 y
+		ARG 5 lines
+		ARG 6 regeneratingHeartIndex
+		ARG 7 maxHealth
+		ARG 8 lastHealth
+		ARG 9 health
+		ARG 10 absorption
+		ARG 11 blinking
```
Comment (haykam821) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping:148]: Some documentation for these arguments would be better.

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 4]
```diff
@@ -126,3 +134,35 @@ CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
...
+	METHOD method_37299 drawHeart (Lnet/minecraft/class_4587;Lnet/minecraft/class_329$class_6411;IIIZZ)V
+		ARG 1 matrices
+		ARG 2 type
+		ARG 3 x
+		ARG 4 y
+		ARG 5 v
+		ARG 6 blinking
+		ARG 7 halfHeart
```
Comment (haykam821) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping:156]: Some documentation for these arguments would be better.

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**